### DISCUSSION TITLE
Option to show more decimal places for numerical habits

### DISCUSSION THREAD

**Discussion by misterbuckley** [iSoron/uhabits#1427] [Feature Requests]
For numerical habits, when I enter a number like "30.05", the app rounds it to 30 and I have to long press on the number to see the exact value. I would like to be able to see the whole value without clicking on it, or have an option to choose how many decimal places to round to. I understand that there's not much space there, so the current behavior of effectively only showing 3 significant digits makes sense in that regard, but it seems to me that, at least on my pixel 6 pro, there would be plenty of room to show 4 or even 5 significant digits.

**Comment by ScalableCodex** [comment]
I agree. I like tracking water and calories and would love to "1255" instead of it rounding all the way up to 1.3k.

**Comment by andreiaribas** [comment]
I agree. I track weekly burn calories and I'd like it to not be rounded. It is rounded even if there's no decimal digit.
